src/main/java/org/minions/devfund/miguelcalderon/WhackAMole.java

```diff
@@ -0,0 +1,200 @@
+package org.minions.devfund.miguelcalderon;
+
+import java.util.Random;
+
+/**
+ * Solution for "Whackmole game" excercise.
+ *
+ * @author Miguel Calderon
+ * @version 1.0 16 Jul 2018
+ */
+
+public class WhackAMole {
+    private int molesLeft;
+    private int attemptsLeft;
+    private char[][] moleGrid;
+    private int score;
+    private static final char MOLE = 'M';
+    private static final char WHACKED_MOLE = 'W';
+    private static final char EMPTY = '*';
+
+    /**
+     * Constructor.
+     *
+     * @param numAttempts   number of attempts.
+     * @param gridDimension number of rows and columns of the grid.
+     */
+    public WhackAMole(int numAttempts, int gridDimension) {
+        this.moleGrid = new char[gridDimension][gridDimension];
+        attemptsLeft = numAttempts;
+        this.molesLeft = 0;
+        score = 0;
+        createEmptyGrid();
+        setRandomPlaceForMoles();
```

Comment:
This is not necessary.
In fact, it will fail your unit tests on edx.
So please remove that method.